scripts/screeps_rl_act_loop_planner.py

```
@@ -0,0 +1,1152 @@
+        current: Any = raw
+        for key in path:
+            if not isinstance(current, dict):
+                current = None
+                break
+            current = lookup(current, (key,))
+        if current is not None:
+            return current
+    return None
+
+
+def named_text(value: Any, aliases: Sequence[str]) -> str | None:
```

Ruff (0.15.13)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)